lib/src/components/NxTabs/types.ts

```diff
@@ -12,12 +12,14 @@ export interface TabContextType {
   rootId: string;
   index: number;
   onTabSelect: (index: number) => void;
+  activationMode?: 'automatic' | 'manual';
 }
 
 export type NxTabsProps = HTMLAttributes<HTMLDivElement> & {
   activeTab?: number | null ;
   onTabSelect: ((index: number) => void);
   children?: ReactNode | null;
+  activationMode?: 'automatic' | 'manual';
```

Comment:
Missing in `nxTabsPropTypes`.  Once added there, I suspect you'll find that you need to add `| null` to this type, and that using default value syntax on the destructuring on https://github.com/sonatype/sonatype-react-shared-components/pull/719/files#diff-c904675f258f76689c48e3ed68a619b63b027ed3b6cb84232531d6e53b37bdf0R24 won't work.